addOns/graphql/CHANGELOG.md

```diff
@@ -3,6 +3,10 @@ All notable changes to this add-on will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+### Fixed
+- NullPointerException behaviour in the VariantGraphQI was fixed (Issue 7689).
+- Additional unit tests were added for the same.
```

Comment:
I appreciate the attention to detail, but this isn't really relevant to end users 😄 I'd say just stick with the first line.

Reply:
Sure. Will change it.